tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py

```diff
@@ -9,8 +9,10 @@
 from supersetapiclient.client import SupersetClient
 
 
-SUPERSET_URL_SCHEME = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}"
-SUPERSET_HOST_URL = f"{SUPERSET_URL_SCHEME}://{{SUPERSET_HOST}}{% if not ENABLE_HTTPS %}:{{ SUPERSET_PORT }}{% endif %}"
+SUPERSET_URL_SCHEME = (
+    "{% if ENABLE_HTTPS and not RUN_SUPERSET %}https{% else %}http{% endif %}"
```

Comment:
Why `not RUN_SUPERSET `?

Reply:
To reduce network traffic, and avoid the TLS resolution